WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -647,7 +647,9 @@ extension MediaLibraryViewController: StockPhotosPickerDelegate {
 
         let mediaCoordinator = MediaCoordinator.shared
         assets.forEach {
-            mediaCoordinator.addMedia(from: $0, to: blog)
+            let info = MediaAnalyticsInfo(origin: .editor(.stockPhotos), selectionMethod: .fullScreenPicker)
+            mediaCoordinator.addMedia(from: $0, to: blog, analyticsInfo: info)
+            WPAnalytics.track(.stockMediaUploaded)
```

Comment:
This one (`WPAnalytics.track(.stockMediaUploaded)`) looks fine there!

The previous comment was about `MediaAnalyticsInfo(origin: .editor(.stockPhotos), selectionMethod: .fullScreenPicker)`.

-- Commenting this just to avoid confusion --